apps/aws-amplify/src/locations/ConfigScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import { useCallback, useState, useEffect, ChangeEvent } from 'react';
+import { ConfigAppSDK } from '@contentful/app-sdk';
+import {
+  Box,
+  Flex,
+  FormControl,
+  Heading,
+  HelpText,
+  Paragraph,
+  Stack,
+  TextInput,
+  TextLink,
+} from '@contentful/f36-components';
+import { css } from 'emotion';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import tokens from '@contentful/f36-tokens';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+import AmplifyIcon, { AWSAmplifyBrand } from '../components/common/AmplifyIcon';
+import { isValidUrl } from '../lib/isVaildUrl';
+
+export interface AppInstallationParameters {
+  amplifyWebhookUrl?: string;
+}
+
+export const styles = {
```

Comment:
Annoying nit: One thing @lilbitner has been pushing for and a pattern we've kind of taken on as a team on the frontend is to separate the styles and components as two separate files unless the styles is dead simple and short

Reply:
No worries, I usually like to keep things flat and co-located as much as tolerable for readability/maintainability but I can definitely support separate the styles into another file